src/core/meta.js

```diff
@@ -26,14 +28,37 @@ module.exports = async function (meta, id) {
 }
 
 const checkRequirements = async (commandRequirements, coreRequirements) => {
+    const OS =  osMap.find(OS => OS.id == os.type());
     for (index = 0; index < commandRequirements.length; index++) {
         try {
             let req = commandRequirements[index];
             let cmdSuccess = await execute(coreRequirements[req], `Checking for ${req}`);
+            if(!cmdSuccess) {
+                print(`Installing ${req}`, "yellow");
+                cmdSuccess = await requirementsInstaller([req]);
+            };
             if (!cmdSuccess) return cmdSuccess;
         } catch (err) {
             return false;
         }
     };
     return true;
+}
+
+const requirementsInstaller = async (requirements) => {
+        const OS =  osMap.find(OS => OS.id == os.type());
+        for (index = 0; index < requirements.length; index++) {
+        try {
+            let req = requirements[index];
+            let cmdSuccess = await execute(OS.installers[req], `Installing ${req}`);
+            if(!cmdSuccess) return cmdSuccess;
+            else {
+                print(`Installed ${req}`, "green");
+                return true;
+            }
+        } catch (err) {
```

Comment:
log error. User needs to know what went wrong.

Reply:
changes pushed